packages/api/src/routes/helpers/provider-route-helper.ts

```diff
@@ -21,6 +21,12 @@ export async function getProviderDataForType<T>(
   const params: DAPIParams = {
     timezoneId: getTimezoneIdFrom("query", req).optional(),
   };
+
+  const rawParams: RawParams = {
+    query: { ...(req.query as RawParams["query"]) },
+    headers: { ...(req.headers as RawParams["headers"]) },
+  };
```

Comment:
We have this in more than one place, good opportunity for a function, `getRawParams(req: Request)` 😃